packages/blocks/src/lib/navigation-map/components/axes-helper.svelte

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
I'm going to probably submit a PR for this to be moved into `@threlte/extras`, but building it here for now just because this repo moves faster